README.md

```
@@ -68,13 +68,79 @@ constants/
+- ~~유효기간 완성 + 에러 없음 → CVC 노출~~
+- ~~CVC 완성 + 에러 없음 → 비밀번호 노출~~
+- ~~비밀번호 완성 + 에러 없음 → 확인 버튼 활성화~~
+- ~~이후 값 수정으로 유효하지 않으면 → 확인 버튼 비활성화~~
+
+7. 확인 버튼 클릭 → /complete 라우팅
+
+- ~~react-router로 페이지 이동~~
+
+8. 카드 등록 완료 페이지 작성
+
+- ~~체크 이미지 + **\*로 시작하는 **카드가 등록되었어요 출력
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)